config/gregtech/worldgen/nether/magnetite_vein2.json

```diff
@@ -1,37 +1,33 @@
-{
-  "weight": 160,
-  "density": 0.3,
-  "max_height": 120,
-  "min_height": 50,
-  "dimension_filter": ["is_nether"],
-  "surface_stone_material": "gold",
-
-  "generator": {
-    "type": "ellipsoid",
-    "radius": [14, 17]
-  },
-  "filler": {
-    "type": "simple",
-    "value": {
-      "type": "weight_random",
-      "values": [
-        {
-          "weight": 60,
-          "value": "ore:magnetite"
-        },
-        {
-          "weight": 15,
-          "value": "ore:iron"
-        },
-		{
-          "weight": 15,
-          "value": "ore:vanadium_magnetite"
-        },
-		{
-          "weight": 10,
-          "value": "ore:precious_metal"
-        }
-      ]
-    }
-  }
+{
+  "weight": 160,
+  "density": 0.3,
+  "max_height": 120,
+  "min_height": 50,
+  "dimension_filter": ["is_nether","dimension_filter:0"],
+  "surface_stone_material": "gold",
+
+  "generator": {
+    "type": "ellipsoid",
+    "radius": [14, 17]
+  },
+  "filler": {
+    "type": "simple",
+    "value": {
+      "type": "weight_random",
+      "values": [
+        {
+          "weight": 60,
+          "value": "ore:magnetite"
+        },
+        {
+          "weight": 15,

```

Comment:
Maybe bump the iron weight to 25 to make up for the loss in gold in the vein? Might not be necessary.